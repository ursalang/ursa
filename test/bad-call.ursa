let h = 3
let g = fn(): Num { h() }
let f = fn(): Num { g() }
f()
