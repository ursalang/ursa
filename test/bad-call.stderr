Error: Line 2, col 21:
  1 | let h = 3
> 2 | let g = fn(): Num { h() }
                          ^~~
  3 | let f = fn(): Num { g() }

Invalid call